kakao/src/main/kotlin/com/agoda/kakao/spinner/helpers.kt

```diff
@@ -0,0 +1,36 @@
+package com.agoda.kakao.spinner
+
+import android.os.Build
+import androidx.test.espresso.Root
+import androidx.test.espresso.matcher.RootMatchers.withDecorView
+import androidx.test.espresso.matcher.ViewMatchers.hasDescendant
+import androidx.test.espresso.matcher.ViewMatchers.withClassName
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.CoreMatchers.allOf
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+
+fun isSpinnerPopup(): Matcher<Root> = SpinnerPopup()
```

Comment:
Could you follow the previous implementation convention for TypeSafeMatcher usage? We don't appreciate to use static functions in our code. 

`isSpinnerPopup` should be a part of Assertion  interface

Reply:
Okay. I've moved it to `SpinnerAdapterAssertions`.